res/skins/Shade/effectunit.xml

```diff
@@ -15,40 +15,54 @@
 				</Style>
 				<BackPath>style/style_bg_effectunit</BackPath>
 				<Children>
-					<WidgetGroup>
-						<Pos>9,5</Pos>
-						<Size>170,90</Size>
-						<Layout>vertical</Layout>
+					<HighlightingGroup>
```

Comment:
I find it confusing that HighlightingGroups are children of ordinary WidgetGroups. I think there should be a different name for the parent, maybe HighlightingGroupContainer?